### PR TITLE
Update authors.yml

### PR DESCRIPTION
**Pull request by Bennykillua**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
 Issue #124

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -155,3 +155,14 @@ siddharth_sable:
...
       instagram: "siddo.exe"
       youtube: "channel/UCf-RytOIE5VugpaPDZdhobA" 
+      
+Benny: 
+  name: "Benny Ifeanyi Iheagwara"
+  role: "Data analyst, technical writer, and speaker" 
+  avatar: "https://raw.githubusercontent.com/Bennykillua/Project/main/Random/Picone.jpg"
+  bio: " Data analyst • Technical writer • Community manager • OSS contributor • Blockchain Enthusiast."
+  links:
+      github: "Bennnykillua"
```
Comment (gouravkhunger) [inline _data/authors.yml:165]: Ah just noticed that you used `Bennnykillua` when your GitHub account is `Bennykillua` (notice the extra `n`).

**Review by gouravkhunger — CHANGES_REQUESTED** [review]
Found the reason why the PR checks failed! Once its corrected it's ready to be merged 🚀

File: _data/authors.yml (hunk 1 above)
Comment (Bennykillua) [reply _data/authors.yml:165]: oh sorry that was a typo

Comment (gouravkhunger) [reply _data/authors.yml:165]: No problem! Make the commit in this branch itself, so you don't need to make new PR again :)

**Review by gouravkhunger — APPROVED** [review]
Great, thanks!

**Pull request merged**